"Hubs" , "Fallback" and "Events"  Icon Update

oddgoo opened the pull request:
-Gave systems "Hubs" and "Events" a short name
-Created Hubs Icon
-Copied over Events icon
-Updated Fallback/Unknown Icon with a better generic "RA" logo
-Increased contrast of the WASM-4 Icon
-renamed PICO-8 icon to lower-case

![image](https://user-images.githubusercontent.com/4585118/233570895-79244b25-4ed7-4a7b-8fd6-1a610d751fb9.png)

![image](https://user-images.githubusercontent.com/4585118/233570940-ff8acccf-afd1-42af-8f8d-72c5496d42d4.png)

![image](https://user-images.githubusercontent.com/4585118/233571293-11b9e280-174c-48bd-85cc-9be91d6a35fd.png)

Jamiras commented:
I think the greyscale RA logo is fine for "All Games", but I don't really like it on the individual pages.

![image](https://user-images.githubusercontent.com/32680403/233681985-3777afb7-df3a-4ae4-ba68-0b21ea3ce6ea.png)

vs the non-description console box:

![image](https://user-images.githubusercontent.com/32680403/233682113-0fb47551-e033-419c-b057-b34108d898b5.png)

oddgoo commented:
> I think the greyscale RA logo is fine for "All Games", but I don't really like it on the individual pages.
> 
> ![image](https://user-images.githubusercontent.com/32680403/233681985-3777afb7-df3a-4ae4-ba68-0b21ea3ce6ea.png)
> 
> vs the non-description console box:
> 
> ![image](https://user-images.githubusercontent.com/32680403/233682113-0fb47551-e033-419c-b057-b34108d898b5.png)
> 

The best solution to this is to complete / add icons to all the non-supported systems on the page that still don't have one. I'm happy to scope that out soon :)

oddgoo commented:
@Jamiras As requested, added placeholders for system so that they can continue using the non-descript black box:

![image](https://user-images.githubusercontent.com/4585118/233880244-40f26180-26ce-47ce-8e7b-39e6af3b8126.png)

Also added a couple more icons that existed in the pack.

![image](https://user-images.githubusercontent.com/4585118/233880322-26d64a98-3afd-4c20-b0a9-442c5955ed3d.png)
